pdf/pdfselection.cpp

```diff
@@ -570,6 +572,34 @@ void PDFSelection::onLayoutChanged()
                      QVector<int>{Rect});
 }
 
+bool PDFSelection::selectionAtPoint(const QPointF &point) const
+{
+    if (d->pageIndexStart < 0 || d->pageIndexStop < 0 ||
+        d->boxIndexStart < 0 || d->boxIndexStop < 0)
+        return false;
```

Comment:
Should have curly braces if conditions wrap multiple lines. || at start of line.